doc/source/uml/mesh.txt

```diff
@@ -3,26 +3,26 @@ abstract class ABCMesh {
   name: str
   data: DataContainer
   --
-  {abstract} add_point(point: Point): uuid.UUID
+  {abstract} add_points(points: iterable of Point): uuid.UUID
   {abstract} get_point(uid: uuid.UUID): Point
-  {abstract} update_point(point: Point)
+  {abstract} update_points(point: iterable of Point)
   {abstract} iter_point(uids: iterable of uuid.UUID {optional}): iterator of Point {unordered}
   ..
-  {abstract} add_edge(edge: Edge): uuid.UUID
+  {abstract} add_edges(edges: iterable of Edge): uuid.UUID
   {abstract} get_edge(uid: uuid.UUID): Edge
-  {abstract} update_edge(edge: Edge)
+  {abstract} update_edges(edge: iterable of Edge)
   {abstract} iter_edge(uids: iterable of uuid.UUID {optional}): iterator of Edge {unordered}
   {abstract} has_edges(): boolean
   ..
-  {abstract} add_face(face: Face): uuid.UUID
+  {abstract} add_faces(faces: iterable of Face): uuid.UUID
   {abstract} get_edge(uid: uuid.UUID): Face
-  {abstract} update_face(face: Face)
+  {abstract} update_faces(face: iterable of Face)
```

Comment:
face -> faces
In general, we need plural in all the methods except add_points and add_edges, which are correct in this PR.
Btw, in the case of particles I named the parameter of the new methods as `iterable`, we have to decide if its ok with iterable or if its better to make the names plural as you did, but I guess we should follow the same naming convention in all the places :+1: